src/components/CVCInput/CVCInput.tsx

```diff
@@ -0,0 +1,31 @@
+import { useCardContext } from "../../contexts/CardContext";
+import { INPUT_CONTAINER } from "../../constants/title";
+import { CARD_VALIDATION_INFO } from "../../constants/CardValidationInfo";
+import InputContainer from "../InputContainer/InputContainer";
+import Input from "../Input/Input";
+
+const CVCInput = () => {
+  const { CVC, CVCHelperText, CVCInputRef, handleCVC } = useCardContext();
+
+  return (
+    <InputContainer title={INPUT_CONTAINER.CVC.TITLE}>
+      <label className="label">CVC</label>
+      <div className="inputContainer">
+        <Input
+          name="cvc"
+          placeholder="123"
+          value={CVC}
+          onChange={handleCVC}
+          ref={(element) => {
+            CVCInputRef.current = element;
+          }}
+          error={CVCHelperText !== ""}
+          maxLength={CARD_VALIDATION_INFO.CVC_MAX_LENGTH}
+        />
+      </div>
```

Comment:
```suggestion
      <label htmlFor="cvc-input" className="label">CVC</label>
      <div className="inputContainer">
        <Input
          id="cvc-input"
          name="cvc"
          placeholder="123"
          value={CVC}
          onChange={handleCVC}
          ref={(element) => {
            CVCInputRef.current = element;
          }}
          error={CVCHelperText !== ""}
          maxLength={CARD_VALIDATION_INFO.CVC_MAX_LENGTH}
        />
      </div>
```
`label` 태그의 `htmlFor` 속성과 `input`태그의 `id` 속성을 매핑해도 좋을 것 같아요~